nautobot/extras/tests/test_api.py

```diff
@@ -1548,6 +1549,11 @@ def test_run_job_no_worker(self, mock_get_worker_count):
         """Job run cannot be requested if Celery is not running."""
         mock_get_worker_count.return_value = 0
         self.add_permissions("extras.run_job")
+        class_path = "api_test_job.APITestJob"
+        job_model = Job.objects.get_for_class_path(class_path)
+        # Make sure no queues are associated with it so it is using the celery default queue.
+        # And the error message is deterministic on line 1573
```

Comment:
Since we're alphabetical now, maybe we should define JobQueueFactory such that it only creates queue names that are alphabetically after "default"? Or can/should we define JobQueue.Meta.ordering somehow (how?) so that 'default' comes first regardless of all other names?

Reply:
Should we define a `models.QuerySet` subclass and define a custom ordering method there?
```
class JobQueueQuerySet(models.QuerySet):
       def order_by_queue_names(self):
             # pop "default" and insert it to the front somehow?

class JobQueue:
       ....
       objects = JobQueueQuerySet.as_manager()
```

Reply:
I actually dont know if the above method is quite possible after some experimenting

Reply:
Yeah I don't think we want to go down that road, but thanks for investigating.